### PR TITLE
Docs: Added a POJO based quick start

### PR DESCRIPTION
**Pull request by rozza**
JAVA-2503

### CODE REVIEW

File: docs/reference/content/driver/getting-started/quick-tour.md [hunk 1]
```diff
@@ -1,418 +0,0 @@
-+++
```
Comment (rozza) [inline docs/reference/content/driver/getting-started/quick-tour.md]: This is no longer needed and should have previously been removed as we have `quick-start.md`

File: docs/reference/content/driver/getting-started/quick-start-pojo.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+import java.util.List;
+
+import static com.mongodb.client.model.Filters.*;
+import static com.mongodb.client.model.Updates.*;
+import static java.util.Arrays.asList;
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+```
+
+- The following Pojo classes. For the full source is available on github for the [Person]({{< srcref "driver/src/examples/tour/Person.java">}}) and [Address]({{< srcref "driver/src/examples/tour/Address.java">}})
```
Comment (jyemin) [inline docs/reference/content/driver/getting-started/quick-start-pojo.md]: Remove "For" from beginning of the sentence.

File: docs/reference/content/driver/getting-started/quick-start-pojo.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+  parent = "MongoDB Driver"
+  identifier = "Sync Quick Start Pojos"
+  weight = 15
+  pre = "<i class='fa'></i>"
++++
+
+# MongoDB Driver Quick Start - Pojos
+
+{{% note %}}
+Pojos stands for Plain Old Java Objects.
```
Comment (jyemin) [inline docs/reference/content/driver/getting-started/quick-start-pojo.md]: Is it convention to not capitalize Pojo in documentation?

File: docs/reference/content/driver/getting-started/quick-start-pojo.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+## Creating a Custom CodecRegistry
+
+Before you can use a Pojo with the driver, you need to configure the [`CodecRegistry` ]({{< relref "bson/codecs.md" >}}) to include a codecs 
+to handle the translation to and from [`bson`]({{< relref "bson/index.md" >}}) for your Pojos. The simplest way to do that is to use the 
+[`PojoCodecProvider.builder()`]({{< apiref "org/bson/codecs/pojo/PojoCodecProvider.html">}}) to create and configure a `CodecProvider`.
+
+The following example will combine a codec registry for the `Person` and `Address` Pojos, with the default codec registry:
+```java
+CodecRegistry pojoCodecRegistry = fromRegistries(
+                fromProviders(PojoCodecProvider.builder().register(Person.class, Address.class).build()),
```
Comment (jyemin) [inline docs/reference/content/driver/getting-started/quick-start-pojo.md]: It's worth showing other way to configure, esp. by package.

File: docs/reference/content/driver/getting-started/quick-start-pojo.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+
+Pass to the methods:
+
+- A filter object to determine the document or documents to update. To facilitate creating filter objects, Java driver provides the [`Filters`]({{< apiref "com/mongodb/client/model/Filters.html">}}) helper. To specify an empty filter (i.e. match all Persons in a collection), use an empty [`Document`]({{< apiref "org/bson/Document.html" >}}) object.
+
+- An update document that specifies the modifications. For a list of the available operators, see [update operators]({{<docsref "reference/operator/update-field">}}).
+
+The update methods return an [`UpdateResult`]({{<apiref "com/mongodb/client/result/UpdateResult.html">}}) which provides information about the operation including the number of documents modified by the update.
+
+### Update a Single Person
```
Comment (jyemin) [inline docs/reference/content/driver/getting-started/quick-start-pojo.md]: Add an example that uses replaceOne as well, showing that you can pass a Person instance directly to the method.

File: docs/reference/content/driver/getting-started/quick-start-pojo.md (hunk 4 above)
Comment (jyemin) [inline docs/reference/content/driver/getting-started/quick-start-pojo.md]: Or is that deferred to the BSON Pojo page?

**Comment by rozza** [comment]
Async copy / code now included.

**Pull request closed without merging**